includes/hub/database/class-event-log.php

```diff
@@ -47,32 +47,39 @@ protected static function get_current_option_name() {
 	 * @return void
 	 */
 	protected static function maybe_update_db() {
-		$db_version = get_option( self::get_current_option_name(), 0 );
+		$db_version = absint( get_option( self::get_current_option_name(), 0 ) );
 		update_option( self::get_current_option_name(), self::DB_VERSION );
```

Comment:
I think you want to also delete this line

Reply:
We still want the db version to be tracked so it runs `dbDelta()` if necessary.

Reply:
Nevermind, I now realize it's a duplicated line